app/mailers/reset_password_mailer.rb

```diff
@@ -4,7 +4,7 @@ def reset_password(email_address, reset_password_code)
     @email_address = email_address
     @reset_password_code = reset_password_code
     mail to: "\"#{email_address.user.full_name}\" <#{email_address.value}>",
-         subject: "Reset your password"
+         subject: "Login instructions"
```

Comment:
Trying to get rid of "Login" in favor of "Sign in".  Can you change to "Instructions for signing in to your OpenStax account"